test/edge.js

```diff
@@ -440,3 +458,66 @@ t.match(
   },
   'should return a minimal human-readable representation of the edge obj'
 )
+
+const bundleChild = {
+  name: 'bundle-child',
+  edgesOut: new Map(),
+  edgesIn: new Set(),
+  explain: () => 'bundleChild explanation',
+  package: { name: 'bundle-child', version: '1.2.3' },
+  get version () {
+    return this.package.version
+  },
+  isTop: false,
+  parent: top,
+  getBundler () {
```

Comment:
I don't think we need `getBundler` in the test if the Edge.bundled getter isn't using it, yes?

Reply:
done